protocols/secio/src/codec/mod.rs

```diff
@@ -103,7 +103,7 @@ impl Hmac {
 }
 
 /// Takes control of `socket`. Returns an object that implements `future::Sink` and
-/// `future::Stream`. The `Stream` and `Sink` produce and accept `BytesMut` objects.
+/// `future::Stream`. The `Stream` and `Sink` produce and accept `Vec<u8>` objects.
```

Comment:
This is obsolete, no?

Reply:
@twittner this change is not within the *plaintext* but within the *secio* module. As far as I can tell `FullCodec` (`EncoderMiddleware` and `DecoderMiddleware`) implement the `Stream` and `Sink` trait with `Vec<u8> and not `BytesMut`. Does that make sense?

Reply:
Ah, sorry. I did not notice that this was in `secio`.